client/components/survey-container/index.tsx

```diff
@@ -1,34 +1,36 @@
-import { useState } from 'react';
 import QuestionStep from './components/question-step';
-import { usePaginatedSurveyContext } from './context';
+import { useSurveyContext } from './context';
 import { Answers, Question } from './types';
 
-type PaginatedSurveyType = {
+type SurveyContainerType = {
 	questions: Question[];
+	answers: Answers;
+	onChange: ( questionKey: string, value: string[] ) => void;
 	recordTracksEvent: ( eventName: string, eventProperties: object ) => void;
 };
 
-const SurveyContainer = ( { questions, recordTracksEvent }: PaginatedSurveyType ) => {
-	const { currentPage, nextPage, previousPage } = usePaginatedSurveyContext();
+const SurveyContainer = ( {
+	questions,
+	answers,
+	onChange,
+	recordTracksEvent,
+}: SurveyContainerType ) => {
+	const { currentPage, nextPage, previousPage, skip } = useSurveyContext();
 	const currentQuestion = questions[ currentPage - 1 ];
```

Comment:
I can see this pattern multiple times in the code. I wonder if it's worth it to have a `currentQuestion` in `useSurveyContext` instead of having `currentPage - 1` in multiple places.

Reply:
Great idea, I'll refactor considering that.